Hassan_Ali/tree.py

```diff
@@ -0,0 +1,67 @@
+
+from multiprocessing.dummy import Array
+
+
+class TreeNode:
+    def __init__(self,value:str,weight:int):
+        self.value = value
+        self.left = None
+        self.right = None
+        self.weight = weight
+# printing using inorder
+# def print_tree(tree):
+#     if not tree:
+#         return
+
+#     print(tree.value,tree.weight)
+#     print_tree(tree.left)
+#     print_tree(tree.right)
+
+def find_path(root:TreeNode,city:str)->str:
+    def helper(node:TreeNode,path:list[str],city:str):
```

Comment:
Use `List` instead of `list`. But you have to import this: `from typing import List`. More details are here https://docs.python.org/3.6/library/typing.html#type-aliases